monai/handlers/validation_handler.py

```diff
@@ -31,14 +31,19 @@ class ValidationHandler:
 
     """
 
-    def __init__(self, interval: int, validator: Evaluator | None = None, epoch_level: bool = True) -> None:
+    def __init__(
+        self, interval: int, validator: Evaluator | None = None, epoch_level: bool = True, exec_at_start: bool = False
+    ) -> None:
         """
         Args:
             interval: do validation every N epochs or every N iterations during training.
             validator: run the validator when trigger validation, suppose to be Evaluator.
                 if None, should call `set_validator()` before training.
             epoch_level: execute validation every N epochs or N iterations.
                 `True` is epoch level, `False` is iteration level.
+            exec_at_start: whether to execute a validation first when starting the training.
```

Comment:
Thanks @Nic-Ma for the PR.

How do we get these metrics? Now `workflow.evaluator.get_stats()` will only give the best metrics right?

Reply:
The initial value of `best metric` is `-1`, so here you can still use `workflow.evaluator.get_stats()` to compare and select.

Thanks.